tasks/configure-agent.yml

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 ---
-- name: Configure the agent with the Stackdriver API key.
+- name: Configure the agent.
   template: src=stackdriver.sysconfig dest={{ stackdriver_sysconfig }} backup=yes
   notify:
```

Comment:
`- restart extractor` throws an error as it is not found. 
Change it to match up with handlers/main.yml: Line 18 

`name: restart extractor if not using gcm` 

or revert it to the original 

`name: restart extractor`